src/main/java/org/terracotta/build/plugins/AngelaPlugin.java

```diff
@@ -116,6 +121,21 @@ public void execute(Task t) {
           } else {
             task.systemProperty("angela.kitInstallationDir", customKitDir.get().getAsFile().getAbsolutePath());
           }
+          CustomHostPluginExtension customHostPluginExtension = (CustomHostPluginExtension) project.getExtensions().getByName("customHostConfig");
+          String customHostNames = customHostPluginExtension.getCustomHosts().getOrNull();
+          StringBuilder spawnedConfigBuilder = new StringBuilder();
+          spawnedConfigBuilder.append("-Xms64m ");
+          spawnedConfigBuilder.append("-Xmx512m ");
+          spawnedConfigBuilder.append("-DIGNITE_UPDATE_NOTIFIER=false ");
+          if (customHostNames != null) {
+            task.systemProperty("angela.additionalLocalHostnames", customHostNames);
+            Path fakeHostFilePath = buildDir.getAsFile().get().toPath().resolve(CustomHostPlugin.CUSTOM_HOST_FILE_NAME);
```

Comment:
This path shouldn't be constructed here. This should be constructed by the CustomHostPlugin... and then maybe used here... or maybe we should just pick it up through the system property?